contracts/strategies/allocation/AllocationWithVotingBase.sol

```diff
@@ -203,7 +211,8 @@ contract AllocationWithVotingBase is IAllocationStrategy, Ownable, Initializable
         }
 
         // Add the application to the applications mapping
-        applications[applicationId] = application;
+        // todo: fix next line
+        // applications[applicationId] = application;
```

Comment:
not clear on why we are commenting this out 🤔

Reply:
the assignment fails with the nested mapping.